style.css

```diff
@@ -490,6 +490,21 @@ hr {
   cursor: pointer;
 }
 
+.Get-In-Touch-btn:hover {
+  cursor: pointer;
+  background-color: white;
+  box-shadow: 0 0 16px rgba(64, 83, 252, 0.24);
+  transform: translateY(4px);
+  color: #6070ff;
+}
```

Comment:
- This is not the right implementation for this hover button. `background-color` should be changed to `#6070ff` color but you have used `white` and `color` should be `white` but you have used `6070ff`.
![image](https://github.com/Nelofarzabi/My_Portfolio/assets/60083437/761682b5-4238-4561-8f11-8aa5d815fae2)
- [ ] Kindly fix this issue to match the given design.